src/testing/utils.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+export const toHaveDirection = (received: Array<ReactTestInstance>) => {
```

Comment:
not sure what this actually does... does it expect the array of ReactTestInstances to all have flex direction `row`? Then the naming should be adjusted.

Reply:
The naming is more generalized. It should check whether the supplied component(s) do have a direction in some way.

Reply:
I'll add a comment